subt/artf_node.py

```diff
@@ -104,12 +131,15 @@ def detect(self, image):
         assert self.width == img.shape[1], (self.width, img.shape[1])
 
         result = self.detector(img)
-        if len(result) > 0:
-            self.stdout(result)
-            report = result2report(result, self.depth)
-            if report is not None:
-                self.publish('artf', report)
-                self.publish('debug_artf', image)  # JPEG
-                self.publish('debug_depth', self.depth)
+        result_cv = self.cv_detector(img)
+        if result or result_cv:
+            self.stdout(result, result_cv)  # publish the results independent to detection validity
```

Comment:
This change actually broke re-run based on logged points - no longer `eval` 
`[('backpack', [(20, 4, 0.9549203), (16, 4, 0.96661633), (24, 8, 0.9785418)])] []`
also the `numpy` type is probably not wanted:
`...(636, 224, 0.9082149)])] [('helmet', 0.6372756, array([614, 224, 637, 246], dtype=int32))]`
or was it expected?

Reply:
> This change actually broke re-run based on logged points - no longer eval
> [('backpack', [(20, 4, 0.9549203), (16, 4, 0.96661633), (24, 8, 0.9785418)])] []

`self.stdout([result, result_cv])` ?

> also the numpy type is probably not wanted:

Ok, the numpy type is not needed there. Should I fix it in the tf_detector and push it to feature/subt-image-xyz?

Reply:
No, leave it as it is - this PR is closed and working, I just needed to write it somewhere last night, sorry ;).
I will discuss it with @zwn and as part of #667 I may log extra streams not needing `eval` ... which would make @zwn happier ...